File: week12_6_14/PRG_단속카메라_민지.py
def solution(routes):
    answer = 1
    routes.sort(key=lambda x: x[1])
    last_cam = routes[0][1]
    for route in routes[1:]:
        if route[0] > last_cam:
            last_cam = route[1]
            answer += 1
    return answer

File: week12_6_14/PRG_단속카메라_민지.java
import java.util.Arrays;
import java.util.Comparator;

class Solution {
    public int solution(int[][] routes) {
        int answer = 1;
		Arrays.sort(routes, Comparator.comparing(o -> o[1]));
		int lastCam = routes[0][1];
		        
        for (int[] route: routes) {
            if (route[0] > lastCam) {
                lastCam = route[1];
                answer++;
            }
        }
        return answer;
    }
}